app/src/main/java/com/example/campusguide/MapsActivity.kt

```diff
@@ -42,6 +44,22 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
             }
         }
 
+        val navigateButton = findViewById<FloatingActionButton>(R.id.navigateButton)
+        navigateButton.setOnClickListener {
+            val getDirectionsDialogFragment =
+                GetDirectionsDialogFragment(
+                    GetDirectionsDialogFragment.DirectionsDialogOptions(
+                        "Concordia University Hall Building",
+                        "Faubourg Building"
```

Comment:
i'd rather these be empty at the start so we can see the placeholder text but it's fine for faster testing rn 👍

Reply:
Yeah, but I think I'll get rid of them so it doesn't cause problems when developing the other stories.